test/proposal.test.ts

```diff
@@ -216,7 +239,26 @@ describe("proposal", function () {
         const totalSupply = await contracts.cowToken.totalSupply();
         expect(
           await contracts.cowToken.balanceOf(contracts.cowDao.address),
-        ).to.equal(totalSupply);
+        ).to.equal(totalSupply.sub(settings.bridge.amountToRelay));
+      });
+
+      it("has approval to send tokens to bridge", async function () {
+        expect(
+          await contracts.cowToken.allowance(
+            gnosisDao.address,
+            multiTokenMediatorETH.address,
+          ),
+        ).to.equal(settings.bridge.amountToRelay);
+      });
```

Comment:
I think it makes sense to create a test contract with the bridge interface that makes the withdraw, instead of using a mock. In this way we see that the approval is working as intended.
Also, it can emit an event with all extra transfer data you might want to test.